__tests__/app.test.js

```diff
@@ -117,3 +117,61 @@ describe('GET/api/articles/:article_id', () => {
       });
   });
 });
+
+describe('GET/api/articles/:article_id/comments', () => {
+  test('status 200, returns an array of comments with correct properties', () => {
+    return request(app)
+      .get('/api/articles/3/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(2);
+        expect(comments[0]).toEqual(
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+          })
+        );
+      });
+  });
+  test('comments are ordered by most recent', () => {
+    return request(app)
+      .get('/api/articles/3/comments')
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy('created_at', { descending: true });
+      });
+  });
+  test('status 404 when article has no comments', () => {
```

Comment:
If an article actually has no comments is this really an error? Nothing has gone wrong necessarily